dot/rpc/modules/author.go

```diff
@@ -94,26 +116,83 @@ func NewAuthorModule(logger log.Logger, coreAPI CoreAPI, txStateAPI TransactionS
 	}
 }
 
-// InsertKey inserts a key into the keystore
-func (am *AuthorModule) InsertKey(r *http.Request, req *KeyInsertRequest, res *KeyInsertResponse) error {
-	keyReq := *req
+// HasSessionKeys checks if the keystore has private keys for the given session public keys.
+func (am *AuthorModule) HasSessionKeys(r *http.Request, req *HasSessionKeyRequest, res *HasSessionKeyResponse) error {
+	pubKeysBytes, err := common.HexToBytes(req.Data)
+	if err != nil {
+		return err
+	}
+
+	pkeys, err := scale.Marshal(pubKeysBytes)
+	if err != nil {
+		am.logger.Debug("err while scale encoding data", "err", err)
```

Comment:
I'd probably remove these logs unless you think they'll be useful in production

Reply:
Removed!